server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java

```diff
@@ -163,7 +165,7 @@ public PlanExecutor() throws QueryProcessException {
 
   @Override
   public QueryDataSet processQuery(PhysicalPlan queryPlan, QueryContext context)
-      throws IOException, StorageEngineException, QueryFilterOptimizationException,
+          throws IOException, StorageEngineException, QueryFilterOptimizationException,
```

Comment:
There are a lot of format changes in this file. I don't think it's necessary.

Reply:
Reformated.